connexion/decorators/validation.py

```diff
@@ -63,6 +64,17 @@ def make_type(value, type_literal):
                 converted = v
             converted_params.append(converted)
         return converted_params
+    elif param_type == 'object':
+        converted_params = {}
+        if 'properties' in param_schema and param_schema['properties']:
```

Comment:
I tried implementing this, and I think this part needs to be recursive also.
I'll try to push up my branch for comparison.

Reply:
i used your suggestion. very nice idea!